README.md

```diff
@@ -19,6 +19,11 @@ A build service can be executed via ```./ta``` or be run via the following go co
 go run cmd/ta/main.go
 ```
 
+The following command will attest a given newline seperated file of Trust Wallet machine IDs to the configured network:
+```
+./ta --attest-machine-ids-by-file keys.yaml
```

Comment:
> a given newline seperated file of Trust Wallet machine IDs

I would rather expect a `csv` or `txt` file instead of yaml. yaml has a very particular syntax it accepts.

Reply:
Further down in the source code you say:
```
fmt.Println("Attestation mode enabled. Using CSV file:", *csvFile)
```
So I would put `keys.csv` into to README